U/Uno/build_tarballs.jl

```diff
@@ -0,0 +1,99 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Uno"
+
+version = v"1.1.0"
+
+sources = [
+    GitSource(
+        "https://github.com/cvanaret/Uno.git",
+        "1f373ce25049e593088bee66b37a9057039ac19a",
+    ),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/Uno
+mkdir -p build
+cd build
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT=blastrampoline-5
+else
+    LBT=blastrampoline
+fi
+
+if [[ "${target}" == *apple* ]] || [[ "${target}" == *freebsd* ]]; then
+    OMP=omp
+else
+    OMP=gomp
+fi
+
+# FortranCInterface_VERIFY fails on macOS, but it's not actually needed for the current build
+sed -i 's/FortranCInterface_VERIFY(CXX)/# FortranCInterface_VERIFY(CXX)/g' ../CMakeLists.txt
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_PREFIX_PATH=${libdir} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -Damplsolver=${libdir}/libasl.${dlext} \
+    -DBLA_VENDOR="libblastrampoline" \
+    -DMUMPS_INCLUDE_DIR=${includedir} \
+    -DMETIS_INCLUDE_DIR=${includedir} \
+    -DMUMPS_LIBRARY="${libdir}/libdmumps.${dlext}" \
+    -DMUMPS_COMMON_LIBRARY="${libdir}/libmumps_common.${dlext}" \
+    -DMUMPS_PORD_LIBRARY="${libdir}/libpord.${dlext}" \
+    -DMUMPS_MPISEQ_LIBRARY="${libdir}/libmpiseq.${dlext}" \
+    -DBLAS_LIBRARIES="${libdir}/lib${LBT}.${dlext}" \
+    -DLAPACK_LIBRARIES="${libdir}/lib${LBT}.${dlext}" \
+    ..
+
+make -j${nproc}
+
+# Uno does not support `make install`. Manually copy for now.
+cp uno_ampl${exeext} ${bindir}/uno_ampl${exeext}
+
+# Currently, Uno does not provide a shared library. THis may bbe useful in future once it has a C API.
+# ${CXX} -shared $(flagon -Wl,--whole-archive) libuno.a $(flagon -Wl,--no-whole-archive) -o "${libdir}/libuno.${dlext}" -L${libdir} -l${OMP} -l${LBT} -ldmumps -lmetis
+"""
+
+platforms = supported_platforms()
+filter!(p -> triplet(p) != "aarch64-unknown-freebsd", platforms)
```

Comment:
Triple comparison is brittle
```suggestion
filter!(p -> !(Sys.isfreebsd(p) && arch(p} == "aarch64"), platforms)
```

Reply:
Oops. Okay. I'll fix in a separate PR.